pkg/tnf/testcases/data/cnf/privilegedpod.go

```diff
@@ -33,7 +33,7 @@ var PrivilegedPodJSON = string(`{
       "name": "HOST_PORT_CHECK",
       "skiptest": true,
       "loop": 1,
-      "command": "oc get pod %s -n %s -o json | jq -r '.spec.containers[%d] | select(.ports) | .ports[].hostPort'",
+      "command": "oc get pod %s -n %s -o go-template='{{range (index .spec.containers %d).ports }}{{if .hostPort}}{{.hostPort}}{{end}}{{end}}'",
```

Comment:
This will work and fix the issue but in case more than one container have hostPorts, they will be printed without separation like: "80808080". Please add an extra espace char between {{.hostPort}} and the following {{end}} to avoid that. I've tested a more complex template to print the pod's and container's name so it can be debugged with the LOG_LEVEL=trace enabled:
```
go-template='{{$putName := .metadata.name}}{{$cut := (index .spec.containers %d)}}{{range $cut.ports }}{{if .hostPort}}PUT {{$putName}} - CUT {{$cut.name}} has declared hostPort {{.hostPort}}{{\"\\n\"}}{{end}}{{end}}'"
```
The output would be:
```
TRACE  [Nov 15 11:15:48.147][spawner.go: 301] STDOUT for /bin/bash  : PUT test-d75b68dfd-dwkh2 - CUT test has declared hostPort 8080
```